tldw_Server_API/app/api/v1/endpoints/slides.py

```
@@ -657,27 +733,19 @@ def _resolve_presentation_visual_style(
 
     resolved_scope = visual_style_scope.strip().lower()
     if resolved_scope == "builtin":
-        builtin = get_builtin_visual_style(resolved_id)
-        if builtin is None:
+        resolved_builtin = resolve_builtin_visual_style(resolved_id)
+        if resolved_builtin is None:
             raise HTTPException(status_code=404, detail="visual_style_not_found")
-        style = _visual_style_response_from_builtin(builtin)
+        return _visual_style_application_from_builtin(resolved_builtin)
     elif resolved_scope == "user":
```

Ruff (0.15.7)

[warning] Unnecessary `elif` after `return` statement

Remove unnecessary `elif`

(RET505)